maintainers/maintainer-list.nix

```diff
@@ -18037,6 +18043,12 @@
     githubId = 11613056;
     name = "Scott Dier";
   };
+  seanrmurphy = {
```

Comment:
@seanrmurphy i added you here and as a maintainer on the wrangler derivation because you put a good bit of work into it, please let me know if you'd rather i remove this or change it.

Reply:
Fine with me - thanks.

Note that I don't think I'm in the maintainers list as yet, so when this PR is complete, it prob needs a specific commit which adds me to the maintainers list.